client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -78,17 +82,37 @@ const granularControlForJetpackStatsCommercialPaywall = [
 	STATS_FEATURE_DOWNLOAD_CSV,
 ];
 
-// Gated modules for WPCOM sites without the FEATURE_STATS_PAID feature.
+// wpcom: Gated modules for WPCOM sites without the FEATURE_STATS_PAID feature.
 const paidStats = [
-	STAT_TYPE_REFERRERS,
-	STAT_TYPE_CLICKS,
 	STAT_TYPE_TOP_AUTHORS,
 	STAT_TYPE_SEARCH_TERMS,
 	STAT_TYPE_VIDEO_PLAYS,
+	STATS_TYPE_DEVICE_STATS,
+	STATS_FEATURE_DATE_CONTROL,
+	STATS_FEATURE_DATE_CONTROL_LAST_30_DAYS,
+	STATS_FEATURE_DATE_CONTROL_LAST_90_DAYS,
+	STATS_FEATURE_DATE_CONTROL_LAST_YEAR,
+	STATS_FEATURE_DOWNLOAD_CSV,
+	STATS_FEATURE_INTERVAL_DROPDOWN_WEEK,
+	STATS_FEATURE_INTERVAL_DROPDOWN_MONTH,
+	STATS_FEATURE_INTERVAL_DROPDOWN_YEAR,
+	STATS_FEATURE_SUMMARY_LINKS_QUARTER,
+	STATS_FEATURE_SUMMARY_LINKS_YEAR,
+	STATS_FEATURE_SUMMARY_LINKS_ALL,
+	STATS_FEATURE_UTM_STATS,
 ];
 
-// Gated controls for WPCOM sites without the FEATURE_STATS_PAID feature.
-const granularControlForPaidStats = [
+// wpcom: Gate UTM and device stats for sites with STATS_FREE feature, this is the feature applied to legacy sites.
+const freeStats = [ STATS_FEATURE_UTM_STATS, STATS_TYPE_DEVICE_STATS ];
```

Comment:
As the feature `stats-free` seems unnecessary, should we change its name to `stats-legacy-free` to highlight the check here?

Reply:
STATS_FREE is a Jetpack site feature I'm re-using here so I don't think the rename makes sense. But maybe using the Jetpack site feature here doesn't make sense either.